spec/unit/pdk/cli/convert_spec.rb

```diff
@@ -11,50 +11,111 @@
 
       expect { PDK::CLI.run(%w[convert]) }.to exit_nonzero
     end
+
+    it 'does not submit the command to analytics' do
+      expect(analytics).not_to receive(:screen_view)
+
+      expect { PDK::CLI.run(%w[convert]) }.to exit_nonzero
+    end
   end
 
   context 'when run from inside a module' do
     before(:each) do
       allow(PDK::Util).to receive(:module_root).and_return('/path/to/test/module')
+      allow(PDK::Module::Convert).to receive(:invoke)
     end
 
     context 'and provided no flags' do
+      after(:each) do
+        PDK::CLI.run(['convert'])
+      end
```

Comment:
Would these be better as `subject` blocks?

Reply:
Not really, as the CLI run isn't the subject of the tests, it's just something that we have to do to trigger the logic being tested.

Reply:
I don't feel super strongly about the `subject` thing but I do think this happening in the `after` hook obfuscates things a bit.